### DISCUSSION TITLE
petty.gq is blocked

### DISCUSSION THREAD

**Discussion by akbolon** [QuiteAFancyEmerald/InvisiProxy#140] [Q&A]
Why is petty.gq blocked?
![Screenshot 2021-05-24 4 24 32 PM](https://user-images.githubusercontent.com/83836072/119403985-2c16fc00-bcad-11eb-9114-5916775c4aeb.png)
And when I try a.petty.gq it has a 404 page?

**Comment by QuiteAFancyEmerald** [comment]
You probably want to read over #44 which explains about obtaining more Holy Unblocker mirrors or sites. The domain you are on was blocked ages ago by Securly globally. 

Also I believe you are mistaking something for that last part of your question. You should be seeing something like the below, not a 404 error page.
Basically you can't access Womginx (or any of the proxies) without authorizing or navigating to it from the main site. 

Page is kinda self explanatory lol

![image](https://user-images.githubusercontent.com/46467239/119426969-0d216580-bcbf-11eb-8782-4b2b5727bab0.png)

**Comment by Susdevlmao** [comment]
Hey bro use stealth more when u are searching stuff press steath if it doesnt load then refresh the page it should work then 
stealth mode hides it i know that my school gets new info abt what i do on my chromebook every month may be the same for yours

**Comment by foxsouns** [comment]
@akbolon securly can be bypassed fairly easily if your school doesnt block you from editing certificates. i am assuming you are on a school chromebook, as that's where securly is enforced for the most part.
1. navigate to `chrome://settings/certificates`
2. find and disable all checkboxes in `org-Amazon/Amazon Root CA 1`.
3. find and disable _everything_ in `org-Starfield Technologies, Inc./`.

if editing those certs are blocked for you, you will need to find a new mirror. read #40.

**Reply by llamatron11** [reply]
unfortunaltly the schools started to block certificates ;(
